releng/release-notes-next/ccache_debug.feature

```diff
@@ -0,0 +1,4 @@
+A new option `debug` has been [added][PR#1408] to the ccache plugin. Setting
+it to `True` creates per-object debug files that are helpful when debugging
+unexpected cache misses.
+https://ccache.dev/manual/4.10.html#config_debug
```

Comment:
You may recycle the same snippet from the previous PR, if you want.  Otherwise, please also update the documentation page, like #1406